### PR TITLE
unsubscan: Add version 0.1.2

### PR DESCRIPTION
**Pull request by LGUG2Z**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by issaclin32** [comment]
Please use 4 spaces for indentation (instead of 2 spaces).

Also, add a new line at the end (otherwise CI will fail).

File: bucket/unsubscan.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "license": "MIT",
+  "architecture": {
+    "64bit": {
+      "url": "https://github.com/LGUG2Z/unsubscan/releases/download/v0.1.2/unsubscan-v0.1.2-x86_64-pc-windows-msvc.zip",
+      "hash": "fb00e0c43336787eeb12151e35bbddf6f0190eded4431bfdcf7776fbda806fdf"
+    }
+  },
+  "bin": [
+    "unsubscan.exe",
+  ],
```
Comment (issaclin32) [inline bucket/unsubscan.json:14]:
```suggestion
  "bin": "unsubscan.exe",
```

**Comment by issaclin32** [comment]
By the way, *VirusTotal* did not detect any virus in it: https://www.virustotal.com/gui/file/fb00e0c43336787eeb12151e35bbddf6f0190eded4431bfdcf7776fbda806fdf

**Comment by LGUG2Z** [comment]
Thanks for the quick feedback! Hopefully I've addressed everything in the last commit. 🤞

File: bucket/unsubscan.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            "url": "https://github.com/LGUG2Z/unsubscan/releases/download/v0.1.2/unsubscan-v0.1.2-x86_64-pc-windows-msvc.zip",
+            "hash": "fb00e0c43336787eeb12151e35bbddf6f0190eded4431bfdcf7776fbda806fdf"
+        }
+    },
+    "bin": "unsubscan.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/LGUG2Z/unsubscan/releases/download/v$version/unsubscan-$version-x86_64-pc-windows-msvc.zip"
```
Comment (issaclin32) [inline bucket/unsubscan.json:17]:
```suggestion
                "url": "https://github.com/LGUG2Z/unsubscan/releases/download/v$version/unsubscan-v$version-x86_64-pc-windows-msvc.zip"
```

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**